app/sprites.py

```diff
@@ -300,28 +344,235 @@ def move(self):
             self.rect.move_ip(-self.speed, 0)
             self.move_collision_out(-self.speed, 0)
 
+    def kill(self) -> None:
+        super().kill()
+        self.engine.score += self.score_points
```

Comment:
You have the same method `kill()` everywhere. It would be better to override this method in a parent instead of all children.